### ISSUE TITLE
Feature request: Add an accessibility option to make text line-breaking in Display Window

### ISSUE THREAD

**Issue by Aleksandra0056** [derceg/explorerplusplus#527]
### Description:

Add an accessibility checkbox option to make text line-breaking in Display Window into `Display Window > Change colours... > Choose font...` page

It would be useful because the text is cut-off, so labels can be seen difficultly.

Defaultly, it should be set as "off" because accessibility features should depend on users, not on app itself.

### Screenshot:

Absence of the setting can be problem for visibility, see:

![Image](https://github.com/user-attachments/assets/d8126fa8-60ba-4f81-9b40-b71ae5477fe1)
